scylla-cql/src/frame/value_tests.rs

```diff
@@ -143,11 +143,6 @@ fn unset_value() {
     assert_eq!(serialized(set_i32), vec![0, 0, 0, 4, 0, 0, 0, 32]);
 }
 
-#[test]
-fn ref_value() {
```

Comment:
Why was the test deleted? Please describe the reason in the commit message.

Reply:
I've reworked the test as you adviced.